src/clientmanager.cpp

```diff
@@ -118,6 +118,10 @@ HSClient* ClientManager::manage_client(Window win, bool visible_already) {
         }
     }
 
+    if (changes.pseudotile.has_value()) {
+        client->pseudotile_ = changes.pseudotile.value();
```

Comment:
There is also a `set_pseudotile` method, which I *suppose* isn't what we want to use here, because it also calls `applyLayout`.

It's also entirely unused. Should it, by any chance, be registered as a setter at the pseudotile attribute?

Reply:
Right, we don't want `set_pseudotile` and I assume we don't need `set_pseudotile` at all. I guess that setting `client->pseudotile_` also calls `applyLayout` because there is a signal connection from the `pseudotile_.changed` signal. Maybe we need an extra `connectSignals()` method in `Client` that is called after the rules are applied.